gcp/modules/locust/swarm.tf

```diff
@@ -41,6 +41,31 @@ resource "null_resource" "locust_swarm_session" {
         RETRY_COUNT=$(($RETRY_COUNT+1))
       done
 
+      # Wait for target service to become rechable from within the locust pod before starting the test
+      RETRIES=100
```

Comment:
100 retries * 10 sec sleep = 1000 sec which 1) is kind of a weird interval (16.666 minutes) and 2) seems like kind of a long time to wait. Maybe 60 retries -> 600s/10m?

Wait, the retry counter resets for each pod, so 3000 sec/50 min in stg/prd. I think it should be shorter.

Reply:
I think I'll need to spend some more time on this issue, generally during my tests I've seen the first pod take a lot of time, the other pods are usually ok by then, so the upper limit is really relevant for the first pod.

I've seen it needing anything from 0-30 retries, therefore quite a high number. Happy to reduce it to smth. like 60.